vpr/src/route/rr_graph2.cpp

```diff
@@ -344,6 +345,7 @@ t_seg_details* alloc_and_load_seg_details(int* max_chan_width,
 
         arch_wire_switch = segment_inf[i].arch_wire_switch;
         arch_opin_switch = segment_inf[i].arch_opin_switch;
+        arch_opin_between_dice_switch = segment_inf[i].arch_opin_between_dice_switch;
```

Comment:
What is the value of arch_opin_between_dice_switch if it is not specified in the architecture file? I assume it should be OPEN (-1). If it is not the case I guess we have to make sure that it has an invalid value (<0)

Reply:
It has a value of -1 if not specified in the arch file.